SwordImportExportPlugin.inc.php

```diff
@@ -119,50 +122,83 @@ public function display($args, $request) {
 					'swordSettings'
 				);
 
-				import('lib.pkp.controllers.list.submissions.SelectSubmissionsListHandler');
-				$selectSubmissionsListHandler = new SelectSubmissionsListHandler(array(
-					'title' 	=> 'navigation.submissions',
-					'count' 	=> 100,
-					'inputName' 	=> 'articleId[]',
-					'getParams' => array(
-						'status'	=> STATUS_PUBLISHED,
-					),
-				));
-				$publishedArticleDao = DAORegistry::getDAO('PublishedArticleDAO');
-				$selectSubmissionsConfig = $selectSubmissionsListHandler->getConfig();
-				$depositedIds = [];
-				foreach ($selectSubmissionsConfig['items'] as $item) {
-					$publishedArticle = $publishedArticleDao->getByArticleId($item['id']);
-					if ($ssi = $publishedArticle->getData("swordStatementIri")) {
-						$depositedIds[$item['id']] = $ssi;
+				if ($this->_version->getMajor() > 2 && $this->_version->getMinor() > 1) {
```

Comment:
Maybe use `version_compare`?

Reply:
Also, feel free to consider how broadly you want this plugin to support different releases of OJS. The more broadly you code, the more spaghetti you'll be maintaining, and I suspect it'll snowball (to mix metaphors). This is my personal preference, but I think it's a better philosophy to release plugins with narrower support and encourage users to keep up to date with their OJS releases. If they can't upgrade, they can use an older version of the plugin. (The Plugin Gallery helps ensure that they know what version is compatible and we encourage its use whenever possible.)